lib/ical_helpers.rb

```diff
@@ -10,13 +19,21 @@
 def generate_ical_items cfg
   ical = Icalendar::Calendar.new
   only_talks(@items).each do |item|
-    event = Icalendar::Event.new
-    event.dtstart = item[:date].to_datetime + (cfg[:unfug_starttime] / 24.0)
-    event.dtend   = item[:date].to_datetime + (cfg[:unfug_endtime]   / 24.0)
-    event.summary = "Unfug"
-    event.description = item[:speakers].join(", ") + " - " + item[:title]
-    ical.add_event event
+    ical.add_event generate_event(item, cfg)
   end
 
   @items << Nanoc::Item.new(ical.to_ical, {:kind => :ical}, "/ical/")
 end
+
+def generate_ical_items_latest cfg
+  latest = cfg[:page][:ical][:latest]
+  return if latest.nil? or latest == false
```

Comment:
You can set the config to `false` to disable the generating of the ical file with the latest entries.